packages/rich-text/src/types.ts

```diff
@@ -0,0 +1,11 @@
+type RichTextFormat = {
+	type: string;
+};
+type RichTextFormatList = Array< RichTextFormat >;
+export type RichTextValue = {
```

Comment:
Here's where we could add some docs for the type. A description should be picked up by our docs script.

Reply:
I tried it, but it looks like I had to put the docs in the root index.js file, since it didn't pick up on the comments in the types.ts file.